meson.build

```diff
@@ -26,6 +26,13 @@ if host_system == 'linux'
   _args += '-Wno-unused-const-variable'
 endif
 
+if cppc.get_id() == 'msvc'
+  add_project_arguments(cppc.get_supported_arguments('/W4'), language : 'cpp')
```

Comment:
Note that meson can set /W4 for you if you use warning_level=3, which for some other compilers sets -Wpedantic so you may or may not want that.

We assume this option always works for the "msvc" compiler, any particular reason you're doing a supported-arguments check here?

Reply:
No, this was basically a straight port of the `CMakeLists.txt` ([here](https://github.com/ERGO-Code/HiGHS/blob/8d71e1ed3744b633da08a235c29f7d194b546ec6/CMakeLists.txt#L266-L275)), but since we do use `warning_level=3` here I'll drop it.